includes/ThemeSupport/Divi.php

```diff
@@ -93,7 +93,7 @@ public function set_current_page( $query, $store_info ) {
 
         if ( $page->ID ) {
             $page->ancestors          = [ $page->ID ];
-            $query->is_singular       = true;
+            $query->is_singular       = false;
```

Comment:
Did you test Divi builder with header and footer on a single store page?

Reply:
Yes, I tested with Divi builder. But header and footer do not work with this change. I have to fix this issue.